### ISSUE TITLE
Formkit CLI export pro inputs

### ISSUE THREAD

**Issue by GionRubitschung** [formkit/formkit#412]
### Describe the feature

With the formkit cli you are currently able to export all the inputs except the pro ones. It would be nice if you could export the pro inputs if in the formkit config a pro key/plugin is active.

**Comment by justin-schroeder** [comment]
Yes! We agree! This is on the roadmap, and we've done all the ground work to allow this in the not-to-distant future. I'll leave this open until we ship that feature.

**Comment by GionRubitschung** [comment]
Very Nice!

**Comment by GionRubitschung** [comment]
Any updates on this?

**Comment by addidesign** [comment]
Is there any updates on this? Also very keen!

**Comment by conradfuhrman** [comment]
Until this is rolled into an update, is there a way to restructure the pro inputs to match the ones we have that are not pro; or a suggestion on best approach there rather than trying to augment the css?

**Comment by GustavoFenilli** [comment]
> Until this is rolled into an update, is there a way to restructure the pro inputs to match the ones we have that are not pro; or a suggestion on best approach there rather than trying to augment the css?

Use a plugin that changes the `:sections-schema` of the pro inputs, that should work.

**Comment by GionRubitschung** [comment]
Any updates? Currently we cannot set the default icon of pro inputs to our own icons because of this... Is there maybe a workaround?

**Comment by GustavoFenilli** [comment]
You can use a plugin and update the `:sections-schema` to update the inputs, not ideal but it is a workaround.
